src/pages/my-certificates/my-certificates.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { useQuery } from '@apollo/client';
+
+import { useStyles } from './my-certificates.styles';
+import { getAllCertificates } from './operations/my-certificates.queries';
+import errorOrLoadingHandler from '../../utils/errorOrLoadingHandler';
+import EmptyCertificates from '../../containers/my-certificates/empty-certificates';
+import FilledCertificates from '../../containers/my-certificates/filled-certificates/filled-certificates';
+
+const MyCertificates = () => {
+  const [certificates, setCertificates] = useState([]);
+  const [count, setCount] = useState([]);
+  const styles = useStyles();
+
+  const { data, loading, error } = useQuery(getAllCertificates, {
+    variables: {
+      limit: 5,
+      skip: 0
+    },
+    onCompleted: () => {
```

Comment:
You should use 'data' that is coming in onCompleted arguments, add one and use it. In another way, you can have bugs.

Reply:
At the beginning I had this way - onComplete: (data), also so this way in other components. But Sonar scolded for a mistake, that I don`t use "data".  When I removed it, Sonar passed well.   
Ok, I`ll return data to onComplete. If Sonar show mistake I`ll inform you